File: number.cpp
#include "stdafx.h"
#include <iostream>
#include <fstream> 
using namespace std;
int main(int argc, char *argv[])
{
	if (argc == 1)
		cout << "Can't find a file" << endl;
	else {
		ifstream infile(argv[1]);
		int n = 0;
		char c;
		while (!infile.eof()) {
			infile.get(c);
			while (c != '\n' && !infile.eof())
				infile.get(c);
			n++;
		}
		cout << "Number of lines" << ' ' << n << endl;
	}

	return 0;
}

File: number.go
package main
import (
"fmt"
"os"
"bufio"
)
func main(){
	if len("os.Args")==1 {fmt.Println("Can't find a file") }
	file,err:=os.Open("os.Args[1]")
	scanner:=bufio.NewScanner(file)
	if err!=nil{
		fmt.Print(err)
	}
	n:=0
	for scanner.Scan(){
		n++
	}
	fmt.Println("number of lines:",n)
}